src/map/skill.cpp

```diff
@@ -9537,6 +9526,9 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		}
 		break;
 
+	case NPC_EMOTION:
+		break;
```

Comment:
I don't think we can move the functionality out like this, certain things only should happen when this skill is used, like how I recently found out it should show "Go!" for angry monsters.

I'm also not sure if for example monsters should drop target if it's not this skill. Should be verified and documented. I always thought the difference between NPC_EMOTION and NPC_EMOTION_ON was that on the latter the monster doesn't drop the target.